release-notes/opensearch-release-notes-2.16.0.md

```diff
@@ -0,0 +1,870 @@
+# OpenSearch and OpenSearch Dashboards 2.16.0 Release Notes
+
+## Release Highlights
+
+OpenSearch 2.16 includes new and updated features to help you build and optimize your search applications, improve stability, availability, and resiliency, enhance ease of use, and more. The release also deprecates CentOS7 as a continuous integration build image and supported operating system.
+
+
+### NEW AND UPDATED FEATURES
+* Fast-filter optimization is now available for general range aggregations, offering the potential for performance improvements of 100x or more as measured against the NOAA workload.
+* OpenSearch now supports byte vector quantization on-cluster as part of your indexing tasks, boosting efficiency for vector compression automation. Support for binary vectors and Hamming distance are also added, unlocking reductions in memory requirements by enabling compression for vectors of up to 32x.
+* A new sort search processor can be configured within a search pipeline to sort search responses, and a new split processor  lets you split strings into arrays of sub-strings. These processors expand the capabilities of search pipelines and add support for more use cases.
+* Updates to the AI connector framework make it possible to integrate any ML model into OpenSearch, allowing you to enable AI enrichments within search flows through the Search API by configuring ML inference search processors.
+* Updates to the cluster manager, including network optimization of cluster manager APIs; compute optimization of pending task processing; and incremental read/writes for routing tables, are designed to reduce the load on the cluster manager. Along with optimizations to shard allocation, these updates can help users scale OpenSearch to more nodes and larger volumes of data.
+* Application-based templates have been added, providing default settings that can simplify the work of tuning your indices for performance of compute and storage resources as well as for usability for common use cases.
+* Support for multiple data sources is extended to two more Dashboards plugins — Notebooks and Snapshot — and all plugins now have version decoupling support in place to filter out incompatible data sources from the selection.
+* In February 2024, OpenSearch issued a deprecation notice regarding CentOS Linux 7, which reached end-of-life on June 30, 2024. As of this release, OpenSearch is deprecating CentOS Linux 7 as a continuous integration build image and supported operating system.
+* The Query Insights plugin is now bundled as a default plugin in the OpenSearch 2.16 distribution. You can use the top n queries APIto identify rogue queries more easily.
+
+### EXPERIMENTAL FEATURES
```

Comment:
Still Pending on adding them, will sync with @jhmcintyre.

Reply:
Just added.

Reply:
@prudhvigodithi  wanted to flag the release highlights are not yet final, pending editorial review. Aiming for completion on this Tuesday morning.